UI/xdg-data/com.obsproject.Studio.appdata.xml.in

```diff
@@ -43,4 +43,7 @@
   <releases>
     <release version="@OBS_VERSION@" date="@APPDATA_RELEASE_DATE@"/>
   </releases>
+  <custom>
+    <value key="flathub::manifest">https://github.com/obsproject/obs-studio/blob/@GIT_HASH@/CI/flatpak/com.obsproject.Studio.json</value>
```

Comment:
```suggestion
    <value key="flathub::manifest">https://github.com/obsproject/obs-studio/blob/@OBS_VERSION@/CI/flatpak/com.obsproject.Studio.json</value>
```

Reply:
That assumes tags are never recreated and releases are never pushed outside the release. Neither are true, realistically.

Reply:
I tend to agree with @barthalion here - tags have been deleted in the past.